_posts/en/newsletters/2018-08-14-newsletter.md

```diff
@@ -35,7 +35,22 @@ projects.
 
 ## Dashboard items
 
-<!-- TODO: @moneyball -->
+{% assign img1_label = "Transactions per block, 25-block moving average, July 14, 2018 - August 13, 2018" %}
+
+- Transaction [fees remain very low][fee metrics]: Anyone who can wait 10 or
+more blocks for confirmation can reasonably pay the default minimum fee rate.
+It’s a good time to [consolidate UTXOs][consolidate info].
+
+- [BTC hash rate][btc hash rate] briefly touched 60 EH/s on August 10, and has a 7 day average of 48 EH/s.
```

Comment:
Dork nit: should say "Estimated BTC hash rate", as there's no way to know what the actual hash rate is in a decentralized network (theoretically, the 60 EH/s could be a realllllly lucky guy CPU mining.

Reply:
Thank you will fix